new-packages.json

```
@@ -134,9 +134,9 @@
   "purescript-halogen-store": "https://github.com/thomashoneyman/purescript-halogen-store.git",
   "purescript-elasticsearch": "https://github.com/ConnorDillon/purescript-elasticsearch.git",
   "purescript-read-generic": "https://github.com/Swordlash/purescript-read-generic.git",
-  "purescript-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
-  "purescript-batteries-env": "https://github.com/purescript-polyform/batteries-env.git",
-  "purescript-batteries-json": "https://github.com/purescript-polyform/batteries-json.git",
-  "purescript-batteries-urlencoded": "https://github.com/purescript-polyform/batteries-urlencoded.git",
+  "purescript-polyform-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
```

The other packages don't have a `bower.json` so they can be renamed - speaking of which, could you add a `bower.json` to them once we merge this change? - but `batteries-core` has one, so we should leave the existing entry in and add a new one instead.

```suggestion
  "purescript-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
  "purescript-polyform-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
```